class-instant-articles-publisher.php

```diff
@@ -99,12 +101,14 @@ public static function submit_article( $post_id, $post ) {
 
 				// Don't process if contains warnings and blocker flag for transformation warnings is turned on.
 				if ( count( $adapter->transformer->getWarnings() ) > 0
-				  && isset( $publishing_settings['block_publish_with_warnings'] )
-					&& $publishing_settings['block_publish_with_warnings'] ) {
+				  && ( ( ! isset( $publishing_settings[ 'publish_with_warnings' ] ) ) || ! $publishing_settings[ 'publish_with_warnings' ] )
```

Comment:
Nit: More consistent to remove the parenthesis enclosing the negated `isset` function?

```
   +---------------------------------------------------------------------------------------------------------------------+
   | +------------------------- remove? --------------------------+                                                      |
   | |                                                            |                                                      |
   v v                                                            v                                                      v
&& ( ( ! isset( $publishing_settings[ 'publish_with_warnings' ] ) ) || ! $publishing_settings[ 'publish_with_warnings' ] )
```